package.json

```diff
@@ -46,6 +46,7 @@
     "moment": "^2.22.2",
     "morgan": "~1.7.0",
     "multer": "^1.3.0",
+    "natives": "^1.1.6",
```

Comment:
What exactly is the reason for using this module? It [sounds unwise](https://www.npmjs.com/package/natives#caveat) to use it.